### DISCUSSION TITLE
Listing 9.11 funktioniert nicht

### DISCUSSION THREAD

**Discussion by TuArHH** [TobiasPSP/OReilly#16] [Fragen] [answered]
Das Listing 9.11 aus dem Buch funktioniert nicht ganz so 

$lottazahlen = 1..49 | Get-Random -Count 6  | Sort-Object
    $Text = $lottazahlen -join '] [ ' 
    "[$Text]"

der erste und der letzte [ ] Klammer fehlt in der Ausgabe.

**Comment by TobiasPSP** [comment, marked as answer]
Sicher? Die kommen doch durch die letzte Zeile, wo die äußeren eckigen Klammern um den Text gestellt werden.
